web/yo/app/views/curationQueue.html

```diff
@@ -0,0 +1,58 @@
+<div>
+    <form ng-submit="addCuration()">
+        <div class="row">
+            <div class="col-sm-5">
+                <input type="text" class="form-control" ng-model="article" placeholder="PMID or Abstract" size="25">
+                <span ng-bind-html="predictedArticle"></span>
+            </div>
+            <div class="col-sm-2"><input type="text" class="form-control" ng-model="link" placeholder="Link" size="10"></div>
+            <div class="col-sm-2"><input type="text" class="form-control" ng-model="variant" placeholder="Variant" size="10"></div>
+            <div class="col-sm-2"><input type="text" class="form-control" ng-model="curator" placeholder="Curator" size="10"></div>
+            <div class="col-sm-1"><button tyle="submit" class="btn btn-success" ng-disabled="!article"><i class="fa fa-plus" style="color:white"></i></button></div>
+        </div>
+    </form>
+    <table datatable="ng" dt-options="dtOptions" dt-columns="dtColumns">
+        <thead>
+        <tr>
+            <th>Article</th>
+            <th>Variant</th>
+            <th>Added by</th>
+            <th>Added at</th>
+            <th>Curator</th>
+            <th>Curated</th>
+            <th>Operation</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr ng-repeat="x in queue">
+            <td>
```

Comment:
Should we consider separate elements into two sections as a whole? One for x.editable and one for !x.editable. Current way, it works. But highly coupled. I would recommend to separate them.